src/includes/performance/distributed-tracing/java.mdx

```diff
@@ -1,7 +1,5 @@
 ## Distributed Tracing
 
-Traces can bridge across multiple software services. Each span in a trace can be represented as a `sentry-trace` header, containing the trace id, span id, and sampling details. This `sentry-trace` header can be passed to downstream services so that they can create spans that are a continuation of the trace started in the originating service.
```

Comment:
same as https://github.com/getsentry/sentry-docs/pull/3363/files#r610391270

use the preview link https://sentry-docs-git-fork-tyrrrz-dotnet-distributed-tracing.sentry.dev/ so you can see how it looks like and spot bugs like that